src/typeOf.ts

```diff
@@ -38,7 +35,7 @@ export function typeOf(obj: any): string {
     if ("Component" in obj && typeOf(obj.Component) === "Svelte") {
       return "SvelteStory";
     }
-    if ("components" in obj && ("template" in obj || "render" in obj)) {
+    if ("__v_isVNode" in obj || "__scopeId" in obj || ("components" in obj && ("template" in obj || "render" in obj))) {
```

Comment:
scopeId allows support of simpler vue story:
```
export const story = () => MyCounter;
```

Reply:
Works from you have 2 stories and you switch from one to the other ?

Reply:
Yes

Reply:
Let's go then